api/build.sh

```diff
@@ -20,6 +20,7 @@ export ENV=local
 pwd=`pwd`
 
 VERSION=$(cat ./api/VERSION)
+GITHASH=$(cat ./.githash 2> /dev/null || git log --pretty=format:"%h" -1)
```

Comment:
There is still a risk of failure. It is recommended to remove the git dependencies.

Reply:
@nic-chen good catch. Thanks. Especially in docker environment, the dependency of `git` command would definitely increase the size of image. Inspired by https://stackoverflow.com/questions/949314/how-to-retrieve-the-hash-for-the-current-commit-in-git , I borrow the awesome script here. Test passed in my local environment. Any feedback is welcome. Thanks.

Reply:
@nic-chen  BTW, for `git` command is already widely used in `Makefile`, so I would prefer to keep using `git` to generate git hash into `.githash`. Please let me know if I was wrong. Thanks.

Reply:
cc @membphis Any ideas?